distribution/docs/src/main/resources/_contents/jconfig.adoc

```diff
@@ -0,0 +1,30 @@
+Version ${project.version}. Copyright (c) Codice Foundation
+:imagesdir: ${project.build.directory}/doc-contents/images
+:toc: left
+:example-caption:
+:source-highlighter: coderay
+:sectlinks:
+:sectanchors:
+:data-uri:
+:sectnums:
+:branding: DDF
+:download-url: https://github.com/codice/ddf/releases
+:adoc-include: ${project.build.directory}/doc-contents/content
```

Comment:
to keep from colliding with the existing docs, many of these files have been prepended with a `j`. This path is slightly different than in the regular `config.adoc`. These would eventually merge once the static docs were fully replaced by the automated ones.